src/skills-builder/test/SkillsBuilder.test.jsx

```diff
@@ -1,19 +1,73 @@
 import { IntlProvider } from '@edx/frontend-platform/i18n';
 import React from 'react';
-import { screen, render } from '@testing-library/react';
+import {
+  screen, render, cleanup, fireEvent,
+} from '@testing-library/react';
+import { mergeConfig } from '@edx/frontend-platform';
 import { SkillsBuilder } from '..';
+import { SkillsBuilderModal } from '../skills-builder-modal';
+import { SkillsBuilderProvider, SkillsBuilderContext } from '../skills-builder-context';
+import { skillsInitialState } from '../data/reducer';
+
+const dispatchMock = jest.fn();
 
 const SkillsBuilderWrapper = () => (
   <IntlProvider locale="en">
-    <SkillsBuilder />
+    <SkillsBuilderProvider>
+      <SkillsBuilder />
+    </SkillsBuilderProvider>
+  </IntlProvider>
+);
+
+const renderSkillsBuilderContext = (value) => (
+  <IntlProvider locale="en">
+    <SkillsBuilderContext.Provider value={[value, dispatchMock]}>
+      <SkillsBuilderModal />
+    </SkillsBuilderContext.Provider>
   </IntlProvider>
 );
 
 describe('skills-builder', () => {
-  it('should render a Skills Builder modal', () => {
+  beforeAll(async () => {
+    await mergeConfig({
+      ALGOLIA_APP_ID: process.env.ALGOLIA_APP_ID || null,
+      ALGOLIA_JOBS_INDEX_NAME: process.env.ALGOLIA_JOBS_INDEX_NAME || null,
+      ALGOLIA_PRODUCT_INDEX_NAME: process.env.ALGOLIA_PRODUCT_INDEX_NAME || null,
+      ALGOLIA_SEARCH_API_KEY: process.env.ALGOLIA_SEARCH_API_KEY || null,
```

Comment:
I don't think we should be trying to use a live service in our tests here.

Reply:
Also, if these values are null, like they would be for most people... would these tests fail?

Reply:
These values have been removed except for `ALGOLIA_JOBS_INDEX_NAME` which has been replaced with a default `test-job-title-index` string